deploy/setup/install-otcetera.sh

```diff
@@ -154,15 +154,15 @@ else
     (
     git clone --recursive https://github.com/OpenTreeOfLife/otcetera
     cd otcetera
-    git branch --track ${otceterabranch} origin/${otceterabranch}
+#    git branch --track ${otceterabranch} origin/${otceterabranch}
```

Comment:
@snacktavish Is this another temporary fix that we should remove? Or do you want to keep?

Reply:
I think we could just use the the 'git_refresh' function, to check out and update the correct branch, but otherwise I think we need this line to check out the right otcetera branch. (or maybe I'm not understanding the q?)

Reply:
Just the usual Q, whether or not this was a hasty post-hack fix that I should restore.